conf/application.conf

```diff
@@ -18,7 +18,7 @@ appName = "employee-expenses-frontend"
 
 play.http.router = prod.Routes
 
-play.filters.csp.CSPFilter = "default-src 'self' 'unsafe-inline' localhost:12345 localhost:9000 localhost:9032 localhost:9250 localhost:9310 *.optimizely.com optimizely.s3.amazonaws.com https://cdn-assets-prod.s3.amazonaws.com www.google-analytics.com www.googletagmanager.com/gtm.js tagmanager.google.com www.googletagmanager.com fonts.googleapis.com tagmanager.google.com ssl.gstatic.com www.gstatic.com fonts.gstatic.com data:"
+play.filters.csp.directives.default-src = "'self' 'unsafe-inline' localhost:12345 localhost:9000 localhost:9032 localhost:9250 localhost:9310 *.optimizely.com optimizely.s3.amazonaws.com https://cdn-assets-prod.s3.amazonaws.com www.google-analytics.com www.googletagmanager.com/gtm.js tagmanager.google.com www.googletagmanager.com fonts.googleapis.com tagmanager.google.com ssl.gstatic.com www.gstatic.com fonts.gstatic.com data:"
```

Comment:
Can we change this from `default-src` to `script-src`. I believe these directives are a 1-to-1 relationship, they don't have a hierarchy like the older way of using CSP, so it doesn't drop down to `default-src` if `script-src` isn't present.